flink-streaming-java/src/main/java/org/apache/flink/streaming/api/transformations/WithBoundedness.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.api.transformations;
+
+import org.apache.flink.annotation.Internal;
+import org.apache.flink.api.connector.source.Boundedness;
+
+/**
+ * An interface to be implemented by transformations that have explicitly set {@link Boundedness}.
+ */
+@Internal
+public interface WithBoundedness {
+
+	/**
+	 * @return The {@link Boundedness} of the implementing entity.
```

Comment:
Maybe this is just a pet peeve but I don't like Javadoc comments that are empty except for a `@return`. I'd just say `Returns the {@link Boundedness} of this {@link Transformation}.`